compile.js

```diff
@@ -120,9 +120,9 @@ function compileFieldRead(ctx, field) {
     case 'bool':     return prefix + 'Boolean()';
     case 'enum':
     case 'uint32':
-    case 'uint64':
+    case 'uint64':   return prefix + 'Varint(false)';
```

Comment:
Should we just use `readVarint()` for more compact code, or will it affect performance?